src/client/particles.cpp

```diff
@@ -149,8 +149,8 @@ void Particle::step(float dtime)
 			m_velocity = p_velocity / BS;
 		}
 	} else {
+		m_pos += (m_velocity + m_acceleration * 0.5f * dtime) * dtime;
```

Comment:
So it's just replacing Euler with Runge-Kutta...?
It's definitely going to be more accurate. We should make this change!